### PR TITLE
Fix portability issue

### PR DESCRIPTION
**Pull request by Rombur**
Make step 1, 2, and 3 ISO 646 compliant. There will be a new requirement by xSDK that all the codes need to be ISO 646 compliant. This is because non-compliant code cannot be run on some of the hardware used by the DOE.

### CODE REVIEW

File: examples/step-1/step-1.cc [hunk 1]
```diff
@@ -15,24 +15,25 @@
 
  */
 
+// clang-format off
 // @sect3{Include files}
 
 // The most fundamental class in the library is the Triangulation class, which
 // is declared here:
-#include <deal.II/grid/tria.h>
+%:include <deal.II/grid/tria.h>
```
Comment (drwells) [inline examples/step-1/step-1.cc:23]: It looks like GitHub's syntax highlighting is not standards-compliant :(

Comment (masterleinad) [reply examples/step-1/step-1.cc:23]: ... and for some reason, `clang-format` messes everything up pretty badly.

Comment (tjhei) [reply examples/step-1/step-1.cc:23]: Well, I vote to remove clang-format and to go back to astyle then.

Comment (tamiko) [reply examples/step-1/step-1.cc:23]: That's a shame. :-(

I thought clang-format would support modern C++ and now it turns out it cannot even deal with digraphs.

Alternatively, I suggest we write a small replacement lexer for clang that we can feed into our own clang-format executable that internally translates between standard syntax and digraphs.

Comment (masterleinad) [reply examples/step-1/step-1.cc:23]: Please open a separate pull request for that.

Comment (tamiko) [reply examples/step-1/step-1.cc:23]: on it.

**Comment by bangerth** [comment]
On 4/1/20 11:46 AM, Matthias Maier wrote:
> Alternatively, I suggest we write a small replacement lexer for clang that we 
> can feed into our own clang-format executable that internally translates 
> between standard syntax and digraphs.

We could use the same kind of hack we use to format the .inst.in files!

**Comment by Rombur** [comment]
It looks like xSDK decided not to go with this requirement after all. So closing this.

**Comment by tamiko** [comment]
@Rombur What a shame. And I was just about to upload and open a pull request for a full bundled llvm/clang with custom lexer.

**Pull request closed without merging**